Gruntfile.js

```diff
@@ -1132,6 +1136,14 @@ module.exports = function(grunt) {
 							BUILD_DIR + 'wp-includes/js/dist/block-editor.js',
 						],
 						dest: BUILD_DIR + 'wp-includes/js/dist/'
+					},
+					{
+						expand: true,
+						flatten: true,
+						src: [
+							BUILD_DIR + 'wp-includes/js/dist/sync.js',
```

Comment:
I'm not sure we should be adding the sync package explicitly to core. See the conversation [here](https://github.com/WordPress/wordpress-develop/pull/5262/files#r1332463531).

The main thing to bear in mind is that, once a package gets added to core, we'll have to support it indefinitely. Given the sync package is in very early stage development, we may be creating a needless back-compat burden here 😬

Reply:
I also commented on it here: https://github.com/WordPress/wordpress-develop/pull/5262#discussion_r1334094989.

Reply:
After the changes made in https://github.com/WordPress/gutenberg/pull/54738, I've updated the npm packages again and `wp-sync` is no longer included in [script-loader-packages.min.php](https://github.com/WordPress/wordpress-develop/pull/5262/commits/670ad3a1cfbb01f56cb167722555440a0d18112e#diff-86b801b80853d4143ddebf95cdd7fed4fe3c20f0617b31fd6d3029549802a2d9). 

Thank you for everyone's help here!